react/Reviews.tsx

```diff
@@ -104,7 +105,7 @@ type ReducerActions =
 
 const initialState = {
   sort: 'ReviewDateTime:desc',
-  from: 1,
+  from: 0,
```

Comment:
Why was this changed? I think this may cause problems in other parts of the code, for example in the SET_NEXT_PAGE / SET_PREV_PAGE reducer actions

Reply:
it has to be 0 because it means the index of all reviews, if we leave this to 1, the first review (index: 0) always will be missing. and i think the pagination can be fixed by adding one like the last change i made on line 646.

Reply:
Ok, yeah, if we change `from` to 0, we also need to change `to` to 9 instead of 10. 
1-10 = 10 reviews
0-10 = 11 reviews 
0-9 = 10 reviews